tests/test_set_config_file.bats

```diff
@@ -0,0 +1,118 @@
+# Source set_config_file function
+. $BATS_TEST_DIRNAME/../shared/bash_remediation_functions/include_lineinfile.sh
```

Comment:
This would need to be updated if the tested is moved to another directory.
CMake could export the source dir variable to be used within the tests, and as a fallback use this relative path to find the remediation file

Reply:
Yes, but why not? If something changes, failing tests themselves will alert the user that they require an update. Tests should IMO be executable easily, without ctest, just by calling `bats`.
The absolute path to remediation functions dir is known at configure-time, but embedding it to the script at that time looks too much of a hassle.

Reply:
I'm ok with the explanation. Solving the conversation.